app/views/events/show.html.erb

```diff
@@ -35,9 +35,11 @@
   </p>
   <br>
   <div class="links_buttoms">
-    <%= link_to t('links.edit'), edit_event_path(@event) %> |
-    <%= link_to t('links.back'), events_path %>
-
-    <%= link_to t('links.show_me_places'), places_path %>
+    <% if can? :manage, @event %>
```

Comment:
It looks for me like you add the same code one more time. You did that before and I think it is on develop. So you should not create the same functionality twice, just rebase to it. This branch is about translations and you have code not relted with translations, mostly with access right.